### PR TITLE
feature: frisby.head()

### PR DESCRIPTION
**Pull request by koooge**
Hello, This is a feature request frisby.head() almost same as #521.

### CODE REVIEW

File: __tests__/fixtures/http_mocks.js [hunk 1]
```diff
@@ -195,6 +195,14 @@ const mocks = {
       });
   },
 
+  head() {
+    return nock(mockHost)
+      .head('/')
+      .reply(200, null, {
```
Comment (H1Gdev) [inline __tests__/fixtures/http_mocks.js]: Why is second argument different from options reply ?

Comment (koooge) [reply __tests__/fixtures/http_mocks.js]: Oh I didn't care it. I will fix

**Comment by koooge** [comment]
Okay, I added a sample to test 'Content-Length'

File: __tests__/nested_spec.js [hunk 2]
```diff
@@ -28,4 +28,20 @@ describe('Frisby nested calls', function() {
...
 
+  // HEAD and GET
+  it('should support http method HEAD', function(doneFn) {
+    mocks.use(['headUsers', 'getUsers']);
+
+    frisby.head(testHost + '/users')
+      .expect('status', 200)
+      .inspectHeaders()
+      .then(function(response) {
+        var length = Number(response.headers.get('Content-Length'));
```
Comment (H1Gdev) [inline __tests__/nested_spec.js]: Please use `let`.

**Comment by vlucas** [comment]
Merged. Thanks! 👍

**Comment by roydekleijn** [comment]
When will this be released?

**Comment by vlucas** [comment]
I made a new release yesterday. Please update to the latest release.

**Pull request merged**